components/formats-bsd/src/loci/formats/out/OMEXMLWriter.java

```diff
@@ -100,6 +102,9 @@ public void setId(String id) throws FormatException, IOException {
       xml = service.getOMEXML(retrieve);
       OMEXMLMetadata noBin = service.createOMEXMLMetadata(xml);
       service.removeBinData(noBin);
+
+      OMEXMLMetadataRoot root = (OMEXMLMetadataRoot) noBin.getRoot();
+      root.setCreator(FormatTools.CREATOR);
```

Comment:
Would it be helpful to append the release version here as well?  Might be useful for diagnostics in the future, if there's ever any version-specific problems.